src/Front/Parser.y

```diff
@@ -243,7 +254,7 @@ matchArms                             --> [(Pat, Expr)]
 
 -- | Each individual arm of a pattern match.
 matchArm                              --> (Pat, Expr)
-  : pat '=>' '{' expr '}'               { ($1, $4) }
+  : pat '=' '{' expr '}'                { ($1, $4) }
```

Comment:
Reverting pattern-match and lambda syntax to use `=` as connective instead of `=>`, for consistency.

Reply:
..consistency with what? this isn't assignment or a binding..